website/docs/r/tg_gateway.html.markdown

```diff
@@ -0,0 +1,52 @@
+---
+layout: "ibm"
+page_title: "IBM : tg_gateway"
+sidebar_current: "docs-ibm-resource-tg-gateway"
+description: |-
+  Manages IBM Transit Gateway.
+---
+
+# ibm\_tg_gateway
+
+Provides a transit gateway resource. This allows transit gateway to be created, and updated and deleted.
+
+## Example Usage
+
+```hcl
+resource "ibm_tg_gateway" "new_tg_gw"{
+name="transit-gateway-1"
+location="us-south"
+global=true
+resource_group=""
+}  
+```
+
+## Argument Reference
+
+The following arguments are supported:
+
+* `name` - (Required, boolean) Name Transit Gateway Services. Example: Transit_Service_BWTN_SJ_DL
+* `location` - (Required, Forces new resource, string) Location of Transit Gateway Services. Example: us-south
+* `global` - (Required, boolean) Allow global routing for a Transit Gateway. If unspecified, the default value is false.
+
+## Attribute Reference
+
+The following attributes are exported:
+
+* `id` - The unique identifier of this gateway. 
+* `name` - The unique user-defined name for this gateway. 
```

Comment:
Remove name,location  ,resource_group from attribute since they are already refereed in Argument reference

Reply:
removed.